README.md

```diff
@@ -10,81 +10,97 @@ in mind so it can be used as more than just GTD software.
 GTG is intended to help you track everything you need to do and need to know,
 from small tasks to large projects.
 
-## Dependencies
+## INSTALLING AND RUNNING
 
-GTG depends on the following packages:
+In order to download and run GTG Developer Version, do the following steps 
+(Debian-based systems):
 
- * Python, version 3.0 or above
- * PyGTK
- * python-support
- * python-xdg
- * python-dbus
- * python-liblarch 
- * yelp (to read GTG documentation)
+DOWNLOAD: 
+Execute this command to get the latest Developer package:
 
-Please refer to your system documentation for information on how to install
-these modules if they're not currently available.
+	$ git clone https://github.com/getting-things-gnome/gtg.git
+
+RUNNING: 
+At first, install python3 dependencies by typing the following command:
+ 
+	$ sudo apt-get install python3-pip
+
+Running of this developer version of GTG will not be possible without installing 
+liblarch at first so clone and install liblarch:
+
+	$ pip3 install -r requirements.txt
+
+RUN:
+In order to run GTG from this Developers repository, you need to launch the debug.sh script:
+
+	$ cd path/to/gtg
+
+	$ ./gtg.sh
+
+Getting Things GNOME launches.
+
+If prompted, you may be required to install also python3-xdg and python3-dbus packages manually. Simply write a command:
+
+	$ sudo apt-get install python3-xdg python3-dbus
+
+Run the script again.
 
-To install the all the required packages providing the basic features on
-Debian-based systems, execute the following command:
-    $ sudo apt-get install python-support python-gtk2 python-gnome2 \
-         python-glade2 python-xdg python-dbus python-liblarch yelp
 
 There are additional plugins (modules for extending the user interface) and
 synchronization services (modules for importing/exporting tasks from/to
 external services) which needs additional packages to work correctly.
 
-### Dependencies for Plugins
+### DEPENDENCIES FOR PLUGINS
 
 "Bugzilla" plugin dependencies:
-  * python-bugz
+  * python3-bugz
 
 "Export and print" plugin dependencies:
-  * python-cheetah
+  * python3-cheetah
   * pdflatex
   * pdftk
   * pdfjam
 
 Installable on Debian-based system via
-    $ sudo apt-get install python-cheetah pdftk pdfjam texlive-latex-base
+    	$ sudo apt-get install python3-cheetah pdftk pdfjam texlive-latex-base
 
 "Geolocalized tasks" plugin is not maintained for a long time and needs to be
```

Comment:
Please drop this section... It will be re-added when geolacalized tasks will be added back